src/bin/sample.rs

```diff
@@ -0,0 +1,13 @@
+use ipc_client::{JsonRpcClientImpl, LotusApi, LotusClient};
```

Comment:
this is example bin, why exclude this?

Reply:
Because it's not something we need in the production code and while it's checked in it's something that we're forced to maintain.

Reply:
We can remove it later? Then how do you expect other to start in the first place?

Reply:
You can add test documentation as comments instead of checking-in unnecessary files just to illustrate their use. See https://doc.rust-lang.org/stable/rustdoc/write-documentation/documentation-tests.html. For a library like this I would avoid having example execution files.